macros/drop_schema.sql

```diff
@@ -0,0 +1,22 @@
+{% macro drop_schema(schema_name, database=None) %}
```

Comment:
Hi Jonathan,
Same comment as in the other PR. You could set `target.database` here as a default value instead of None, and line 5 could then be deleted.